src/packagedcode/gemspec.py

```diff
@@ -0,0 +1,265 @@
+#
+# nexB Inc. and others. All rights reserved.
+# http://nexb.com and https://github.com/nexB/scancode-toolkit/
+# The ScanCode software is licensed under the Apache License version 2.0.
+# Data generated with ScanCode require an acknowledgment.
+# ScanCode is a trademark of nexB Inc.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# When you publish or redistribute any data created with ScanCode or any ScanCode
+# derivative work, you must accompany this data with the following acknowledgment:
+#
+#  Generated with ScanCode and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  ScanCode should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  ScanCode is a free software code scanning tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/scancode-toolkit/ for support and download.
+
+
+import csv
+from collections import OrderedDict
+import io
+import re
+import os
+import glob
+
+
+#!/usr/bin/env python
+#
+# Copyright 2015 Balasankar C <balasankarc@autistici.org>
```

Comment:
Is this really GPL? if so we cannot use it in ScanCode as this not compatible with the Apache license. 
AFAIK @balasankarc kindly relicensed it to a choice of MIT or GPL on my request  https://github.com/balasankarc/gemfileparser/issues/4
So we would likely pick the MIT choice instead. @balasankarc can you make this a tad more explicit in the code comment header?
Also I wonder if we could instead contribute and further @balasankarc library and use it as a dep rather than forking it here.

@balasankarc what's your plan for https://github.com/balasankarc/gemfileparser long term? We could take over its maintenance if you like?

Reply:
I agree with @pombredanne.

Reply:
@rpotter12 We want to use `gemfileparser` as a library instead of forking its source code here.

Reply:
@majurg we can't use https://github.com/balasankarc/gemfileparser library directly as it can only extract dependencies info only. So we have edited the existing library. I think @pombredanne wants to say that we can fork https://github.com/balasankarc/gemfileparser in @nexB and add our edited code and release a new library version so that we can extract all the data and use it in our library.

Reply:
@pombredanne https://github.com/balasankarc/gemfileparser is in dual license - GPLv3 and MIT.